.dockerignore

```diff
@@ -0,0 +1,2 @@
+venv/**/*
+venv
```

Comment:
I do this too ... but best to include newlines... I think somewhere you can get vscode to ping you about this.
(Or `echo "venv" >> .dockerignore` does the trick, I think.

Reply:
If I'm interpreting this correctly, you want me to have a newline at the end of the file? You got it